src/perception/tag_detector/tag_detector.processing.cpp

```diff
@@ -128,7 +131,13 @@ namespace mrover {
         }
 
         if (mPublishImages && mImgPub.getNumSubscribers()) {
-            cv::aruco::drawDetectedMarkers(mImg, mCorners, mIds);
+
+            cv::aruco::drawDetectedMarkers(mImg, mImmediateCorners, mImmediateIds);
+            for (auto& [id, tag]: mTags) {
+                cv::Scalar color(255, 0, 0);
+                cv::Point pt(1200, 100);
```

Comment:
I don't think we want to hardcode this. While it works with 1280x720 images that could change in the future.
I also think that we should put the tag ID in here as well not just the hit hount.
If there are multiple tags too this will try to draw them in the same exact spot.
I would suggest making a counter then using that integer to offset the y-value.

Reply:
Fine for now